src/atomate2/vasp/flows/lobster.py

```diff
@@ -0,0 +1,194 @@
+"""Flows for Lobster computations."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker
+from pymatgen.core import Structure
+
+from atomate2.vasp.flows.core import DoubleRelaxMaker
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.jobs.core import RelaxMaker, StaticMaker
+from atomate2.vasp.jobs.lobster import (
+    VaspLobsterMaker,
+    delete_lobster_wavecar,
+    get_basis_infos,
+    get_lobster_jobs,
+    update_user_incar_settings_maker,
+)
+from atomate2.vasp.sets.core import StaticSetGenerator
+
+__all__ = ["LobsterMaker"]
+
+
+@dataclass
+class LobsterMaker(Maker):
+    """
+    Maker to perform a Lobster computation
+
+    Optional optimization.
+    Optional static computation with symmetry
+    static computation with ISYM=0 is performed
+    several Lobster computations testing several basis sets is performed
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    bulk_relax_maker : .BaseVaspMaker or None
+        A maker to perform a relaxation on the bulk.
+        Set to ``None`` to skip the
+        bulk relaxation
+    additional_static_run_maker: .BaseVaspMaker or None
+        A maker to perform a preconvergence run
+        before the wavefunction computation without symmetry
+    vasp_lobster_maker : .BaseVaspMaker
+        A maker to perform the computation of the wavefunction before the static run.
+        Cannot be skipped.
+    code: str
+        determines the dft code. currently only vasp is implemented.
+        This keyword might enable the implementation of other codes
+        in the future
+    calculation_type:
+        "standard" for a normal Lobster run
+    delete_all_wavecars:
+        if true, all WAVECARs will be deleated after the run
+    user_lobsterin_settings:
+        Dict including additional information
+    user_supplied_basis: dict or None
+        dict to supply a basis for the lobsterrun
+    additional_outputs:
+        list of filenames to include in output (str)
```

Comment:
See previous comment about these fields. My alternative suggestion would be to replace these with a `pure_lobster_maker` field instead which would hold the fields instead of the flow maker. Again, might not suitable in this case but please consider it.